pkg/skaffold/docker/image_test.go

```diff
@@ -227,6 +227,13 @@ func TestGetBuildArgs(t *testing.T) {
 			},
 			want: []string{"--network", "bridge"},
 		},
+		{
+			description: "no-cache",
+			artifact: &latest.DockerArtifact{
+				NoCache: "noCache",
```

Comment:
Since we made the value a bool, this should now be set to true

```go
NoCache: true
```

Reply:
Should be good now.